0009-endpoint-events-ipc/metadata-updated.avsc

```diff
@@ -33,28 +33,69 @@
       "doc": "The application version name the endpoint registered in."
     },
     {
-      "name": "removed",
-      "type": {
-        "type": "array",
-        "items": "string"
+      "name":"removed",
+      "type":{
+        "type": "map",
+        "values":{
+          "type": "record",
+          "namespace": "org.kaaproject.events.endpoint.lifecycle.gen.v1",
+          "name": "MetadataKeyRemovedDto",
+          "fields": [
+            {
+              "name": "oldValue",
+              "type": ["null", "string"],
+              "default": null
+            }
+          ],
+          "doc" : "DTO with oldValue JSON string that was removed."
```

Comment:
Do not document the the DTO classes. 
It may me not clear to the reader what is `DTO`.
It will be enough to document `oldValue` field with something like 
`The JSON value of metadata key that was removed. null if metadata key was associated with null JSON value`